command/pr.go

```diff
@@ -101,13 +112,20 @@ func prStatus(cmd *cobra.Command, args []string) error {
 	fmt.Fprintln(out, "")
 
 	printHeader(out, "Current branch")
-	if prPayload.CurrentPR != nil {
-		printPrs(out, 0, *prPayload.CurrentPR)
+	currentPR := prPayload.CurrentPR
+	currentBranch, _ := ctx.Branch()
+	remoteRepo, _ := remoteContext.BaseRepo()
```

Comment:
why not use the `baseRepo` variable from line 81? my reading of this is that it's doing the same work as `determineBaseRepo`.

Reply:
`baseRepo` is of type `ghrepo.Interface` which only had `RepoName` & `RepoOwner` strings whereas this `remoteRepo` is of type `api.Repository` which has the `DefaultBranchRef` struct I needed

I'm still pretty new to golang so I could be being dumb about this though

Reply:
not dumb! golang is confusing here and i don't think we're making the best use of interfaces to begin with.

but, check out type assertions: https://tour.golang.org/methods/15 . `baseRepo.(*api.Repository)`, for example, gives you access to the struct members otherwise obscured by baseRepo being typed with a generic interface.

Reply:
@vilmibm Thank you for the link! I've switched to using type assertions instead